pkg/kubectl/cmd/rollout/rollout_undo.go

```diff
@@ -112,9 +112,7 @@ func (o *UndoOptions) CompleteUndo(f *cmdutil.Factory, cmd *cobra.Command, out i
 		Flatten().
 		Do()
 	err = r.Err()
-	if err != nil {
-		return err
-	}
+	cmdutil.CheckErr(err)
```

Comment:
checkErr should be called in the caller. The caller needs to stop appending errors.